source/_integrations/tuya.markdown

```diff
@@ -25,79 +24,119 @@ ha_platforms:
 ha_dhcp: true
 ---
 
-The `tuya` integration is the main integration to integrate [Tuya Smart](https://www.tuya.com) related platforms, except the Zigbee and Bluetooth(BLE) hub. This includes devices linked with the Tuya, Smart Life, and Jinvoo Smart apps. You will need your account information (username, password and account country code) from one of these apps to discover and control devices which related to your account.
+The Tuya Home Assistant integration is developed for controlling **Powered by Tuya (PBT)** devices using the [tuya-iot-python-sdk](https://github.com/tuya/tuya-iot-python-sdk) (a Python version of [Tuya Open API](https://developer.tuya.com/en/docs/iot/api-reference?id=Ka7qb7vhber64)), which is officially maintained by the Tuya Developer Team.
 
-**Important**: Not all Tuya devices are supported by the `tuya API` used by this integration. For more details refer to [TuyaHA Library](https://github.com/PaulAnnekov/tuyaha).
+It applies to smart devices that you have added to the Tuya Smart or Smart Life app.
 
-There is currently support for the following device types within Home Assistant:
+## Supported device categories
+- **Climate**: Supports air conditioner, heater, thermostat.
+- **Cover**: Supports curtain, curtain switch.
+- **Fan**: Supports fan, air purifier, humidifier, dehumidifier.
+- **Light**: Supports light, light strip, ambient light, light string, humidifier's light, Ceiling light.
+- **Scene**: Supports smart scene.
+- **Switch**: Supports switch, socket, power strip, smart kettle, breaker, pet water feeder, air purifier, diffuser.
 
-- **Climate** - The platform supports the air conditioner and heater.
-- **Cover** - The platform supports curtains.
-- **Fan** - The platform supports most kinds of Tuya fans.
-- **Light** - The platform supports most kinds of Tuya light.
-- **Scene** - The device state in frontend panel will not change immediately after you activate a scene.
-- **Switch** - The platform supports switch and socket.
+## Quick installation and startup
 
-{% include integrations/config_flow.md %}
+### Configure on the Tuya IoT Platform
+
+#### Prerequisites
+
+You have created an account for [Tuya Smart or Smart Life app](https://developer.tuya.com/en/docs/iot/tuya-smart-app-smart-life-app-advantages?id=K989rqa49rluq#title-1-Download).
+
+#### Create a project
+
+1. Log in to the [Tuya IoT Platform](https://iot.tuya.com/).
+2. In the left navigation bar, click **Cloud** > **Projects**.
+3. On the page that appears, click **Create**.
+4. In the **Create Project** dialog box, configure **Project Name**, **Description**, **Industry**, and **Availability Zone**. In the **Development Method** field, select **Smart Home PaaS** from the dropdown list.
+	> **Note:** Select all available zones.
+
+	![](https://airtake-public-data-1254153901.cos.ap-shanghai.myqcloud.com/content-platform/hestia/1622030427d285b460b84.png)
```

Comment:
All images need to be included in the PR and hosted on home-assistant.io. No external URLs for images.

Reply:
replace with local images in images folder.